vs-code-home-page-sample/main.bal

```diff
@@ -0,0 +1,88 @@
+import ballerina/http;
+
+type Person record {
+    string id;
+    string firstName;
+    string lastName;
+    int age;
+    string country;
+};
+
+type Course record {
+    readonly string id;
+    string name;
+    int credits;
+};
+
+type Student record {
+    string id;
+    string fullName;
+    string age;
+    record {
+        string title;
+        int credits;
+    }[] courses;
+    int totalCredits;
+    string visaType;
+};
+
+const D_TIER_4_VISA = "D tier-4";
+
+var totalCredits = function(int total, record {string id; string name; int credits;} course) returns int => total + (course.id.startsWith("CS") ? course.credits : 0);
+
+function transform(Person person, Course[] courses) returns Student => let var isForeign = person.country != "LK" in {
```

Comment:
Lets use this in the service.

Reply:
Ok 👍🏼

Reply:
Fixed via [20e1fb0](https://github.com/ballerina-guides/integration-samples/pull/27/commits/20e1fb0de69931807521ff4a7a8344bdf46a5a14)